easycv/datasets/ocr/data_sources/ocr_rec_datasource.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) Alibaba, Inc. and its affiliates.
+import logging
+import os
+import traceback
+
+import cv2
+import lmdb
+import numpy as np
+
+from easycv.datasets.registry import DATASOURCES
+from easycv.file.image import load_image
+
+
+@DATASOURCES.register_module(force=True)
+class OCRRecSource(object):
```

Comment:
does ocr recognition support pai labeling format?

Reply:
only detection algo support，recognition algo need crop text from original image for training